tests/converter/rasa.py

```diff
@@ -0,0 +1,57 @@
+"""This module tests the convertion tool that is used to convert a rasa or diaglogflow
+project into a mindmeld project"""
+
+from mindmeld import NaturalLanguageProcessor
+from mindmeld.converter.rasa_converter import RasaConverter
+
+
+def test_rasa_converter(rasa_project_path, mindmeld_project_path):
+    converter = RasaConverter(rasa_project_path, mindmeld_project_path)
+    converter.convert_project()
+
+
+def test_nlp_greet(mindmeld_project_path):
```

Comment:
Instead of building nlp component per function, you can write a fixture

```
@pytest.fixture
def rasa_nlp(mindmeld_project_path):
    nlp = NaturalLanguageProcessor(mindmeld_project_path)
    nlp.build()
    return nlp
```

```
def test_nlp_greet(rasa_nlp):
    query1 = 'hi there'
    assert nlp.process(query1)['intent'] == 'greet'

Reply:
Oh ok cool. So you pass in the nlp into each test?